whois-nrtm4-client/src/test/java/net/ripe/db/nrtm4/client/processor/UNFProcessorWrongSignatureTestIntegration.java

```diff
@@ -0,0 +1,35 @@
+package net.ripe.db.nrtm4.client.processor;
+
+import net.ripe.db.nrtm4.client.AbstractNrtmClientIntegrationTest;
+import net.ripe.db.nrtm4.client.dao.NrtmClientVersionInfo;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@Tag("IntegrationTest")
+public class UNFProcessorWrongSignatureTestIntegration extends AbstractNrtmClientIntegrationTest {
+
+    @BeforeAll
+    public static void setUp(){
+        System.setProperty("nrtm.key", "{\"kty\":\"OKP\",\"crv\":\"Ed25519\",\"kid\":\"35e18000-ce6a-4b72-b92c-d09fc6ec279d\",\"x\":\"ZqHS2UmPulPoWlPIET-nSuXyEkb4N-LKiWZMSj8S2eZ\"}");
```

Comment:
Instead of property may be use the  file, as we need to rotate the key , in that case property will not work

Reply:
Done